elyra/pipeline/processor_airflow.py

```diff
@@ -251,7 +251,7 @@ def create_pipeline_file(self, pipeline, pipeline_export_format, pipeline_export
                                                                      name=pipeline.runtime_config)
             user_namespace = runtime_configuration.metadata.get('user_namespace') or 'default'
 
-            description = f"Created with Elyra {__version__} pipeline editor using {pipeline.name}.pipeline."
+            description = f"Created with Elyra {__version__} pipeline editor using {pipeline.source}."
```

Comment:
Minor improvement (no impact on functionality) - we surround file names with tick marks to make the messages easier to read. As is, one might see something awkward, like `[...] using my file.ipynb.`

```suggestion
            description = f"Created with Elyra {__version__} pipeline editor using '{pipeline.source}'."
```